blobstore/src/main/java/org/jclouds/blobstore/TransientStorageStrategy.java

```diff
@@ -83,7 +85,9 @@ public boolean containerExists(final String containerName) {
 
    @Override
    public Iterable<String> getAllContainerNames() {
-      return containerToBlobs.keySet();
+      ArrayList<String> containers = new ArrayList<String>(containerToBlobs.keySet());
+      Collections.sort(containers);
+      return containers;
```

Comment:
I checked Atmos, Azure, CloudFiles, and AWS-S3 and these all sort as well.  Can you sort in `FilesystemStorageStrategyImpl.getAllContainerNames` for consistency?